.github/workflows/test-package.yml

```diff
@@ -0,0 +1,97 @@
+name: Test the release package
+
+on:
+  # TODO: Remove push trigger before merge.
+  push:
+  workflow_dispatch:
+    inputs:
+      emscripten_repository:
+        description: 'Emscripten repository'
+        required: true
+        default: 'Windows-on-ARM-Experiments/emscripten'
+        type: string
+      emscripten_branch:
+        description: 'Emscripten repository branch'
+        required: true
+        default: 'fix-unit-tests-arm64'
+        type: string
+      emscripten_test_suite:
+        description: 'Emscripten test suite'
+        required: true
+        default: 'test_hello_world'
+        type: string
+
+defaults:
+  run:
+    working-directory: test-package
+
+jobs:
+  bootstrap-workflow:
+    runs-on: [ self-hosted, Windows, ARM64, WASM ]
+    timeout-minutes: 10
+
+    steps:
+      - name: Create working directory
+        working-directory: ${{ github.workspace }}
+        run: New-Item -ItemType Directory -Force -Path test-package
+
+      - name: Git checkout
+        uses: actions/checkout@v3
+        with:
+          path: test-package
+
+  install-package:
+    needs: bootstrap-workflow
```

Comment:
Now only one runner is used, however jobs should be independent

Reply:
Do I understand correctly that you are suggesting that each job should clone it's own version of emsdk repository to a different directory?

Reply:
Correct, maybe it makes sense to join them into one job.

Reply:
Ignore my previous comment, I thought you are commenting on a different workflow.

Anyway, I've refactored the wrokflow to a single job.